components/atoms/Input/Input.js

```diff
@@ -0,0 +1,22 @@
+import React, { useCallback, useRef, useEffect } from 'react';
+
+const Input = ({ value, onChange, autoFocus, ...props }) => {
```

Comment:
Add please destructing default values, especially for `onChange`, because if this props will be `undefined`, call of them throw an error